features/facebookImport/src/model/caching.js

```diff
@@ -0,0 +1,80 @@
+import EntityDeserializer from "./entities/utils/entity-serialisation";
+
+const namespace = "http://polypoly.coop/schema/fbImport/";
+
+export default class ExportDataCache {
+    constructor(pod, fileId) {
+        this._pod = pod;
+        this._fileId = fileId;
+    }
+
+    _createAccessQuery(dataFactory) {
+        return {
+            subject: dataFactory.namedNode(`${this._fileId}/cache`),
+            predicate: dataFactory.namedNode(`${namespace}importerCache`),
+        };
+    }
+
+    async _selectDataQuad(dataFactory, polyIn) {
+        return await polyIn.select(this._createAccessQuery(dataFactory));
+    }
+
+    _createDataQuad(dataFactory, rawData) {
```

Comment:
I'm not totally sure using the triple store for what's simply a key-value pair will be the most efficient way of doing it. What other alternatives have you considered?